src/OSPSuite.Engine/Domain/DataFactory.cs

```diff
@@ -27,15 +27,6 @@ public interface IDataFactory
       /// <returns></returns>
       DataRepository CreateRepository(IModelCoreSimulation simulation, SimModelNET.ISimulation simModelSimulation, string repositoryName = null);
 
-      /// <summary>
-      ///    Update the molweight for the given datacolumn representing the quantity given as parameter.
-      ///    Molweight is at most available for observers and molecule amount. If the molecular weight is not found,
-      ///    the default value will be set as is (null)
-      /// </summary>
-      /// <param name="column">Column whose molweight should be updated</param>
-      /// <param name="quantity">Quantity whose values are saved in the column</param>
-      /// <param name="model">Model where the quantity belongs</param>
-      void UpdateMolWeight(DataColumn column, IQuantity quantity, IModel model);
```

Comment:
@Yuri05 I moved this guy because it's required in PKSim and MoBi and has nothing to do with SimModel or funcparser